src/PrestaShopBundle/Service/Form/MultistoreCheckboxEnabler.php

```diff
@@ -75,24 +76,32 @@ class MultistoreCheckboxEnabler
      */
     private $multistoreController;
 
+    /**
+     * @var FormCloner
+     */
+    private $formCloner;
+
     /**
      * MultistoreCheckboxEnabler constructor.
      *
      * @param FeatureInterface $multistoreFeature
      * @param ShopConfigurationInterface $configuration
      * @param Context $multiStoreContext
      * @param MultistoreController $multistoreController
+     * @param FormCloner $formCloner
      */
     public function __construct(
         FeatureInterface $multistoreFeature,
         ShopConfigurationInterface $configuration,
         Context $multiStoreContext,
-        MultistoreController $multistoreController
+        MultistoreController $multistoreController,
```

Comment:
Injecting a controller to reuse a behavior is not necessarily the best idea. But the point is out of scope here.

Reply:
yes it's not ideal...

 the `configurationDropdown` method from this controller returns the dropdown, it needs informations about the current configuration value and related configuration class, the dropdown is attached to the field so we add it in the options so it can be accessed in the template

it's definitely outside of the scope of this PR but if you have an idea for improving this don't hesitate to open an issue about it 👍

Reply:
I was indeed thinking about a common service, with optional templating because who knows ?
Definitely out of scope, and I might handle it in a future PR indeed :)